rancid-tomatillos/src/Components/Header/Header.js

```diff
@@ -1,23 +1,39 @@
 import React from 'react';
 import './Header.scss';
 import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { removeUser, updateLoggedIn } from '../../actions/index';
 
 
-const Header = ({ data }) => {
+const Header = ({ data, isLoggedIn, removeUser, updateLoggedIn }) => {
+  let log;
+  if (isLoggedIn) {
+    log = <button onClick={() => {updateLoggedIn(isLoggedIn); removeUser();}} className='header-btn'>Log Out</button>
```

Comment:
This is awesome! it nice job working through how to call two functions in an onClick event.